### DISCUSSION TITLE
v0.4.2

### DISCUSSION THREAD

**Discussion by tomaztk** [tomaztk/SQLServer-Data-Lineage#4] [Announcements]
The initial release (minor changes) of T-SQL Data lineage script for SQL Server databases (SQL Server, Azure SQL server, Azure Database, Azure Edge, Azure MI, and Azure Synapse)

<hr /><em>This discussion was created from the release <a href='https://github.com/tomaztk/SQLServer-Data-Lineage/releases/tag/sql-server'>v0.4.2</a>.</em>

**Comment by LuukMo** [comment]
Hey, 
Very nice project. 
Found a small bug (I think). If you remove the last carriage return from the test query it won't return the last object.

**Reply by tomaztk** [reply]
Hi,
Thank you for your effort, this is great.
Please fork the repository and commit the changes. 

Best, Tomaz
